### ISSUE TITLE
NullPointerException: Cannot get state after load - On project importing

### ISSUE THREAD

**Issue by manuc66** [hsz/idea-gitignore#327]
Hi,

I got the following exception when importing the project https://github.com/syncany/syncany :

```
Cannot get state after load
java.lang.NullPointerException
	at mobi.hsz.idea.gitignore.settings.IgnoreSettings.getState(IgnoreSettings.java:160)
	at mobi.hsz.idea.gitignore.settings.IgnoreSettings.getState(IgnoreSettings.java:52)
	at com.intellij.configurationStore.StateGetter.close(StorageBaseEx.kt:52)
	at com.intellij.configurationStore.ComponentStoreImpl.a(ComponentStoreImpl.kt:324)
	at com.intellij.configurationStore.ComponentStoreImpl.a(ComponentStoreImpl.kt:409)
	at com.intellij.configurationStore.ComponentStoreImpl.reinitComponents(ComponentStoreImpl.kt:452)
	at org.jetbrains.settingsRepository.SyncKt$updateStoragesFromStreamProvider$3.invoke(sync.kt:186)
	at org.jetbrains.settingsRepository.SyncKt$updateStoragesFromStreamProvider$3.invoke(sync.kt)
	at com.intellij.openapi.application.ActionsKt$invokeAndWaitIfNeed$2.run(actions.kt:42)
	at com.intellij.openapi.application.impl.ApplicationImpl.invokeAndWait(ApplicationImpl.java:650)
	at com.intellij.openapi.application.ActionsKt.invokeAndWaitIfNeed(actions.kt:42)
	at com.intellij.openapi.application.ActionsKt.invokeAndWaitIfNeed$default(actions.kt:28)
	at org.jetbrains.settingsRepository.SyncKt.updateStoragesFromStreamProvider(sync.kt:169)
	at org.jetbrains.settingsRepository.SyncKt.updateStoragesFromStreamProvider$default(sync.kt:143)
	at org.jetbrains.settingsRepository.AutoSyncManager$sync$$inlined$catchAndLog$lambda$1$1.run(autoSync.kt:151)
	at com.intellij.openapi.application.TransactionGuardImpl$6.run(TransactionGuardImpl.java:318)
	at com.intellij.openapi.application.impl.LaterInvocator$1.run(LaterInvocator.java:166)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.a(LaterInvocator.java:417)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:401)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:756)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:709)
	at java.awt.EventQueue$3.run(EventQueue.java:703)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:726)
	at com.intellij.ide.IdeEventQueue.b(IdeEventQueue.java:843)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:679)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:391)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)
```

Thanks

**Comment by spacecrio** [comment]
Same

**Comment by Hdzi** [comment]
Same

**Comment by kraluk** [comment]
Same here. Even if I've reactivated it after a crush.

**Comment by hsz** [comment]
Thank you for the report. It has been already reported and fixed with #315
